x/blob/keeper/keeper_test.go

```diff
@@ -101,6 +101,7 @@ func CreateKeeper(t *testing.T) (*keeper.Keeper, store.CommitMultiStore, sdk.Con
 		cdc,
 		paramsSubspace,
 	)
+	// change the params to a non default value
```

Comment:
this comment doesn't make sense to me b/c the next line sets the params to the default values.

```suggestion
```